CMSgov/easi-app
# EASI-2413 Trb admin initial request form

# EASI-2413

[Figma](https://www.figma.com/file/iGatgTnlzTuqW9CQFYHDam/TRB-Intake-e2e?node-id=205%3A16091)

## Changes and Description

Trb admin view of the Initial Request form: `InitialRequestForm.tsx`

After some spec checking with @aterstriep, we decided that a few items from the original ticket that will **not** be in this pr. They will be broken out in upcoming trb pr's.
- "Is this request ready box" has its own ticket
- "Notes" to the top right will be in a different pr



## How to test this change

Fill out a request, note it's uuid, and check it out with this path: `/trb/uuid/initial-request-form`
